experimental/generation/generator/templates/standard/date-clear.dialog.lg

```diff
@@ -0,0 +1,8 @@
+[generator.lg](../generator.lg)
+
+# filename
+- ${clearFilename()}
+
+# template
+- ${clearDialog()}
```

Comment:
All of the files in this directory need to move to packages/library/templates/standard